### PR TITLE
Feature/support android12

### PR DESCRIPTION
**Pull request by Guidemarcus**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Bugfix
- Feature
- Code style update (formatting)
- Refactoring (no functional changes, no api changes)
- Build or CI related changes
- Documentation content changes
- Other... Please describe:

## PR Checklist 
Please check if your PR fulfills the following requirements:

- [ ] Interface members are XML documented
- [ ] Documentation (XML or comments) has been added and/or existing documentation has been updated
- [ ] [Architecture documents](./doc/Architecture.md) have been updated
- [ ] Tested on all relevant platforms

### CODE REVIEW

File: src/app/ApplicationTemplate.UWP/Views/Styles/Application/Brushes.xaml [hunk 1]
```diff
@@ -6,6 +6,10 @@
 					xmlns:ios="http://nventive.com/ios"
 					mc:Ignorable="xamarin android ios">
 
+	<ResourceDictionary.MergedDictionaries>
+		<ResourceDictionary Source="MaterialOverride.xaml" />
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.UWP/Views/Styles/Application/Brushes.xaml:10]: I don't think we want to merge MaterialOverride.xaml more than once.
@MatFillion, @kazo0

**Comment by jeanplevesque** [comment]
I don't understand how adding support for Android 12 changes 50 files.

**Comment by Guidemarcus** [comment]
This PR will be aborted. I worked on this branch for too long and found too many refactoring tasks and instabilities. I will make multiple PR based on my commits @jeanplevesque

File: src/app/ApplicationTemplate.UWP/Views/Styles/Application/Brushes.xaml (hunk 1 above)
Comment (MatFillion) [reply src/app/ApplicationTemplate.UWP/Views/Styles/Application/Brushes.xaml]: Correct that probably doesn't work.

**Comment by MatFillion** [comment]
@Guidemarcus Yeah, let's definitely do the uno update first, then another PR for CI image update, then Android12. Plenty of small PRs will be easier 🚀

File: src/app/ApplicationTemplate.UWP/Views/Content/DadJokes/DadJokesPage.xaml [hunk 2]
```diff
@@ -35,9 +35,9 @@
 					x:Uid="DadJokesPage_CommandBar">
 			<CommandBar.PrimaryCommands>
 				<AppBarButton Command="{Binding NavigateToFilters}">
-					<PathIcon Data="M10 18H14V16H10V18ZM3 6V8H21V6H3ZM6 13H18V11H6V13Z"
-							  HorizontalAlignment="Center"
-							  Foreground="{StaticResource MaterialOnPrimaryColor}" />
```
Comment (carlh98) [inline src/app/ApplicationTemplate.UWP/Views/Content/DadJokes/DadJokesPage.xaml:40]: I think we need to keep this for Dark mode purposes

**Pull request closed without merging**